### DISCUSSION TITLE
My nation's rights problem

### DISCUSSION THREAD

**Discussion by Sylar40** [crunchy-labs/crunchy-cli#423] [Q&A]
Hello. First, I'd like to thank the creator for this fantastic tool, which is amazing for someone like me. However, I am facing a very big problem. Regarding the anime of One Piece and Naruto, in my country, Crunchyroll broadcasts them in 4:3 640x480 format. However, the preview on the navigation bar shows me a 16:9 1920x1080 format. How can I download, with my (Italian) account, the original video format? Preserving the subtitles in my language? 
I use this ep for example:
crunchy-cli archive -o "S{season_number}E{episode_number} - {title}.mkv" -r best -a ja-JP -s it-IT https://www.crunchyroll.com/it/watch/GRWEX4QMR/homecoming

![Ep 01](https://github.com/crunchy-labs/crunchy-cli/assets/145484372/8b21b3ca-ead7-4ac9-85b3-451742e53fb9)

![Screenshot 2024-05-21 000818](https://github.com/crunchy-labs/crunchy-cli/assets/145484372/79b9ee99-426f-4ad7-bc8b-1dd300f2b28f)

Thank you very much :D

**Comment by bytedream** [comment]
The navigation bar previews are pre-generated independently from the actual video by Crunchyroll.
You can think of the preview like an image that contains multiple frames. When downloaded and split up into its separate frames, you can see that it only consists of a couple frames that are more or less 4:3:
![image](https://github.com/crunchy-labs/crunchy-cli/assets/63594396/7490cf70-7e9c-424b-b102-59145415235d)

**Reply by Sylar40** [reply]
Hello bytedream, thank you for the quick response. Can you confirm if you also have this format in 4:3? Because the episodes in 16:9, which are much more HD than these, do exist. My alternative is to download the srt files and then embed them in other video files. Unfortunately, I am a fan of the original audio, but EVERYONE only uses hardsubs on their videos. Thanks again :)
